blog/_posts/btcpay-server-2-0.md

```diff
@@ -0,0 +1,157 @@
+
+## 🔑 Key features
+
+- New interface localization
+- New sidebar-only navigation
+- New onboarding flow for new users and Point Of Sale
+- Three new e-commerce integrations (Wix, Odoo & BigCommerce)
+- Checkout v2 default (depreciation of the legacy checkout)
+- Branding enhancements
+
+Full changelog can be found [here](https://github.com/btcpayserver/btcpayserver/releases/tag/v2.0.0).
+
+## ⚠️ 2.0 breaking changes
+With BTCPay Server 2.0, we’re introducing breaking changes in our API that allow us to get rid of technical debt that accumulated over the years. While these changes are necessary to future-proof BTCPay and enable exciting new features, we understand that they may affect some users. 
+
+For most, the transition to 2.0 will be seamless, but if you rely on **custom integrations or plugins**, you will need to update them to ensure compatibility with our 2.0 API. 
+
+We’ve done our best to make this process as straightforward as possible, and we strongly recommend updating your integrations immediately after the BTCPay Server update. If you're unsure whether these changes will impact you, please check our [breaking changes documentation](https://github.com/btcpayserver/btcpayserver/issues/5964) for detailed guidance.
+
+BTCPay Server started in 2017, and over the past seven years, we’ve learned a lot as a community. New protocols have emerged, and users have continuously requested new features. Throughout this journey, our main challenge in product development has been to move slowly and steadily, ensuring we avoid breaking changes for our users. Stability is critical because it allows merchants to focus on running their businesses without worrying about infrastructure or payment integrations. 
+
+Commitment to stability meant that BTCPay evolved organically, always maintaining backward compatibility. While this benefits users, it has placed increasing pressure on our developers and plugin creators, making it harder and slower to meet new expectations, a challenge known as the "technical debt tax"; that we will now eliminate.
+
+During the transition from 1.0 to 2.0, larger instances may experience **a few minutes of database migration**. Once this initial migration is complete, BTCPay will be fully operational, though it will continue migrating some data in the background. For instance, some invoices may not immediately appear in reports or searches, but newer data is migrated first, so we expect minimal disruption. 
+
+As an example, our mainnet demo database, containing around 400,000 invoices, took one to two minutes to start and about 15-20 minutes to fully update all invoices in the background.
+
+## 🔄 Updating to 2.0
+
+// TODO 
+
```

Comment:
We need to update this section upon we sync cc @NicolasDorier @rockstardev

Reply:
Sure thing, I'll be back in action tomorrow morning; here is proposed section text we can discuss:

```suggestion
Updating to version BTCPay Server 2.0 is a one-way process with no option for rolling back. Because of that, we're making it opt-in, requiring that you [SSH into your server](https://docs.btcpayserver.org/FAQ/ServerSettings/#how-to-ssh-into-my-btcpay-running-on-vps) and run two simple commands:

- git fetch && git checkout 2.0
- btcpay-update.sh

Please note that the database migration might take a few minutes, depending on the size of your installation. Enjoy BTCPay 2.0 and let us know your thoughts!
```